lava_test_plans/projects/lkft-android/cts-lkft.yaml

```diff
@@ -0,0 +1,27 @@
+{% extends "testcases/android-cts.yaml" %}
+
+{% set test_timeout = 480 %}
+{% set test_name = "cts-lkft" %}
+{% set cts_test_plan = "cts-lkft" %}
```

Comment:
what makes this a cts-lkft test and not just a regular cts test?

Reply:
non lkft-android project does not need to run this job. or say they might have their plans on what test modules to be run.